tests/storage-volumes-vm

```diff
@@ -118,22 +129,22 @@ do
 	waitInstanceReady v2
 
 	# shellcheck disable=2016
-	lxc exec v1 -- /bin/sh -c 'mount /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_block-dev1 /mnt && [ $(cat /mnt/bar) = foo ] && umount /mnt'
+	lxc exec v1 -- /bin/sh -c 'mount /dev/disk/by-id/scsi-0QEMU_QEMU_HARDDISK_lxd_vol2 /mnt && [ $(cat /mnt/bar) = foo ] && umount /mnt'
```

Comment:
These lines have changed multiple times in the same PR - please dont do this.

Each commit in this PR should be the final version of that logical change.

Additionally these line changes do not relate to the commit message:

>  tests/storage-volumes-vm: Check links to /dev/s*

Reply:
Sorry, this one was an accident during the last rebase, fixed